.ibm/pipelines/lib/k8s-wait.sh

```diff
@@ -0,0 +1,240 @@
+#!/usr/bin/env bash
+
+# Kubernetes/OpenShift resource waiting and polling utilities
+# Dependencies: oc, kubectl, lib/log.sh
+
+set -euo pipefail
```

Comment:
This `set` will be carried over to the whole script and affect the whole execution. 
I'd keep this set up only on the entry point (openshift-ci-tests.sh at the moment) and refrain from redefining it **or** the `set` in the library should be aligned with the `set` in the entrypoint.